### PR TITLE
Working/lb/190609

### PR DESCRIPTION
**Pull request by louberger**
ip: fixup references
ip: untab and whitespace cleanup to keep xml2rfc happy
ip: revise aggregation section, add alloc to sec 6

### CODE REVIEW

**Review by dfedyk — COMMENTED** [review]
Reviewed changes

**Review by dfedyk — APPROVED** [review]

File: ip/draft-ietf-detnet-ip.xml [hunk 1]
```diff
@@ -572,79 +571,48 @@ DetNet                    |L2/SbN|          |L2/SbN|
...
+          From a single node perspective, the aggregation of IP flows
+          impacts DetNet IP data plane flow identification and resource
+          allocation.  As discussed above, IP flow identification uses
+          the IP "6-tuple" for flow identification.  DetNet IP flows can
+          be aggregated using any of the 6-tuple fields defined in <xref
+          target="ip-flow-id"/>.  The use of prefixes, wildcards,
+          bitmasks, and value ranges allows a DetNet node to identify
+          aggregate DetNet flows.  From a resource allocation
+          perspective, DetNet nodes must provide service to a aggregate
+          and not on a component flow basis.
```
Comment (vb-balazs) [inline ip/draft-ietf-detnet-ip.xml:603]: Terminology: individual flows vs, component flows? Do we need a new term (component flow)?

File: ip/draft-ietf-detnet-ip.xml [hunk 2]
```diff
@@ -935,19 +902,25 @@ DetNet                    |L2/SbN|          |L2/SbN|
...
-            UDP Destination Port value.
+            providing a different service for the aggregate of all other
+            flows with that same UDP Destination Port value.
+          </t>
+          <t>
+            It is the responsibility of the DetNet controller plane to
+            properly provision both flow identification information and
+            the flow specific resources needed to provided the traffic
+            treatment needed to meet each flow's service requirements.
+            This applies for aggregated and non-aggregated flows.
```
Comment (vb-balazs) [inline ip/draft-ietf-detnet-ip.xml:913]: Again terminology: non-aggregated flow = individual flow. Do we need a new term?

File: mpls/draft-ietf-detnet-mpls.xml [hunk 3]
```diff
@@ -1221,18 +1189,38 @@ DFx = DetNet member flow x over a TE LSP
...
+     for forwarding sub-layer aware nodes:
      <list style="symbols">
-       <t>MPLS Service Label and d-CW labels at a DetNet end system.</t>
-       <t>MPLS Service Label and d-CW in the context of one or more MPLS Forwarding Labels at any DetNet aware device along the path.</t>
-       <t>Any aggregate flow by one or more MPLS Forwarding Labels at any DetNet aware device along the path.</t>
-       <t>Any other DetNet flow information for example IP Flow Identification beyond the MPLS service label as specified in
-       <xref target="I-D.ietf-detnet-ip"/> at a DetNet aware device. </t> 
+       <t>Each DetNet F-Label value and associated traffic parameters.</t>
+       <t>F-Label to H-LSP/Label mappings.</t>
+       <t>H-Label values and service parameters.</t>
```
Comment (vb-balazs) [inline mpls/draft-ietf-detnet-mpls.xml:1212]: H-label?

File: mpls/draft-ietf-detnet-mpls.xml [hunk 4]
```diff
@@ -1221,18 +1189,38 @@ DFx = DetNet member flow x over a TE LSP
...
-     identify an individual DetNet flow in an MPLS context:
+     for forwarding sub-layer aware nodes:
      <list style="symbols">
-       <t>MPLS Service Label and d-CW labels at a DetNet end system.</t>
-       <t>MPLS Service Label and d-CW in the context of one or more MPLS Forwarding Labels at any DetNet aware device along the path.</t>
-       <t>Any aggregate flow by one or more MPLS Forwarding Labels at any DetNet aware device along the path.</t>
-       <t>Any other DetNet flow information for example IP Flow Identification beyond the MPLS service label as specified in
-       <xref target="I-D.ietf-detnet-ip"/> at a DetNet aware device. </t> 
+       <t>Each DetNet F-Label value and associated traffic parameters.</t>
+       <t>F-Label to H-LSP/Label mappings.</t>
```
Comment (vb-balazs) [inline mpls/draft-ietf-detnet-mpls.xml:1211]: Is this for seamless MPLS? What about SR?

**Review by vb-balazs — COMMENTED** [review]
Let's discuss these comments

**Pull request merged**